> LINK : fatal error LNK1104: cannot open file 'papilo\binaries\papilo.exp'

~Where is this `.exp` coming from? It's not standard, no?~
EDIT: yes it seems to be a Windows thing.

--- a/recipe/bld.bat
+++ b/recipe/bld.bat
@@ -23,7 +23,7 @@ cmake -G Ninja ^
       %CMAKE_ARGS%
 if %ERRORLEVEL% neq 0 exit 1
 
-cmake --build scipoptsuite-build --parallel "%CPU_COUNT%"
+cmake --build scipoptsuite-build
 if %ERRORLEVEL% neq 0 exit 1
 
 cmake --install scipoptsuite-build --prefix "%LIBRARY_PREFIX%"

